### ISSUE TITLE
Dash Bio apps need to be moved to dash-bio.plotly.host

### ISSUE THREAD

**Issue by shammamah** [plotly/dash-bio#133]
In accordance with https://github.com/plotly/streambed/issues/12263, we are making our `index.py` app obsolete. The applications we currently have are not true standalone apps, since they just modify the layout on the main `index.py` app. Right now, `index.py` lives on that server (https://dash-gallery.plotly.host/dash-bio) and its portal (https://dash-bio.plotly.host/Portal/)

What we need to do: 
- [x] Publish the dash bio package to test.pypi.org 
- [x] Separate the requirements for each app from our `requirements.txt` file, add in the `dash_bio` dependency
- [ ] Set up a file structure within the repository that will house all of the applications. I'm thinking just a folder called `demo_apps` that contains a folder for each app 
- [ ] Copy each app's relevant `sample_data` files into its folder
- [ ] Create all the necessary files (looks like we need a `config.py` and `runtime.txt`) for each app in its folder
- [ ] Once the apps are on the portal, tag them (this looks like a really cool feature we should take advantage of!) with things like "genomics", "visualizations", etc. 

Let's get this done before I fly out to Boston for training with Amgen (13 Feb)! They're a huge pharmaceutical company and it would be awesome to show off Dash Bio and sell them on DDS simultaneously :) let's get that 🍞

**Comment by shammamah** [comment]
@mkcor @Bachibouzouk We can easily convert each `app_[]` file to a standalone app by adding the following lines: 

```
app = dash.Dash()
app.scripts.config.serve_locally = True
server = app.server
app.layout = layout() # assign layout 
app.callbacks() # register all of the callbacks 
``` 
somewhere in the beginning of the file and 

```
if __name__ == '__main__':
    app.run_server(debug=True)
``` 

at the end. Will try this out with one of our applications and confirm :)

**Comment by Bachibouzouk** [comment]
@shammamah : I did 
```
import dash


def run_standalone_app(layout, callbacks, port=8050, debug=True):

    app = dash.Dash()
    app.scripts.config.serve_locally = True
    app.config["suppress_callback_exceptions"] = True

    server = app.server

    app.layout = layout()  # assign layout
    callbacks(app)  # register all of the callbacks
    app.run_server(debug=debug, port=port)

```
in a `app_standalone.py` file in `tests/dashbio_demos/utils`, then 

```
if __name__ == '__main__':
    from tests.dashbio_demos.utils.app_standalone import run_standalone_app
    run_standalone_app(layout, callbacks)
```
at the end of an app file works (for me at least).
Then the css needs to be linked properly :)

**Comment by shammamah** [comment]
@Bachibouzouk Great idea -- then all we really need to change is the `runtime.txt` file for each application :)

**Comment by mkcor** [comment]
Ok! @Bachibouzouk Do you have a work-in-progress branch for this, that I should piggyback on? If not, I'll create one.

**Comment by Bachibouzouk** [comment]
@mkcor I don't have a branch for this yet, you can create it :)

**Comment by mkcor** [comment]
Aww unrelated but `app_molecule3d.py` must have been written on Windows because I'm stuck in this nightmare https://stackoverflow.com/questions/64749/m-character-at-end-of-lines :(

**Comment by shammamah** [comment]
@mkcor We need to expose the variable for the apps to run on the server. So it will work locally, but it will break once it's uploaded :)

**Comment by mkcor** [comment]
@shammamah Oh, of course it works *locally*!! haha
Will edit ;)

**Comment by Bachibouzouk** [comment]
@shammamah : is this issue still open?

**Comment by shammamah** [comment]
@Bachibouzouk It shouldn't be! Fixed in #149 -- must have forgotten to close it :)
